sqldelight-gradle-plugin/src/test/migration-success-noisolation/build.gradle

```diff
@@ -0,0 +1,36 @@
+buildscript {
+  apply from: "${projectDir.absolutePath}/../buildscript.gradle"
+}
+
+apply plugin: 'org.jetbrains.kotlin.jvm'
+apply plugin: 'com.squareup.sqldelight'
+
+repositories {
+  maven {
+    url "file://${projectDir.absolutePath}/../../../../build/localMaven"
+  }
+  mavenCentral()
+}
+
+sqldelight {
+  Database {
+    packageName = "com.example"
+    verifyMigrations = true
+  }
+}
+
+tasks.withType(com.squareup.sqldelight.gradle.VerifyMigrationTask) {
+  useClassLoaderIsolation = false
+}
+
+tasks.withType(com.squareup.sqldelight.gradle.GenerateSchemaTask) {
+  useClassLoaderIsolation = false
+}
+
+tasks.withType(com.squareup.sqldelight.gradle.SqlDelightTask) {
+  useClassLoaderIsolation = false
+}
+
+tasks.withType(com.squareup.sqldelight.gradle.GenerateMigrationOutputTask) {
+  useClassLoaderIsolation = false
+}
```

Comment:
@AlecStrong I get the following with these:
```
   > Could not set unknown property 'useClassLoaderIsolation' for task ':verifyMainDatabaseMigration' of type com.squareup.sqldelight.gradle.VerifyMigrationTask.
```

Reply:
is it property shit, like do you have to call `useClassLoaderIsolation`

i've never actually exposed properties on a task here so i'm as noob as you

Reply:
I believe you have to set classloaderIsolation on the work that you submit to the WorkerExecutor. Check the Gradle 5.6 release notes for an example: https://docs.gradle.org/5.6/release-notes.html

Reply:
Turns out it was just a local IntelliJ configuration issue. Once I fixed that, it passed there + passes CI and local CLI